app/scripts/app.js

```diff
@@ -131,3 +131,21 @@ var app = angular.module('trunkApp', ['ngRoute', 'mm.foundation', 'highcharts-ng
   var Viewer = {
     'Scene': null
   };
+
+
+app.directive('autocomplete', function() {
```

Comment:
Ogromnie dużo tu pustych linii jest. Co prawda to ja zawsze powtarzam, że za nie się nie płaci, ale nie widzę sensu mieć ich aż tyle :)